decidim-initiatives/app/controllers/decidim/initiatives/create_initiative_controller.rb

```diff
@@ -16,6 +16,7 @@ class CreateInitiativeController < Decidim::Initiatives::ApplicationController
 
       helper Decidim::Admin::IconLinkHelper
       helper InitiativeHelper
+      helper SignatureTypeOptionsHelper
       helper_method :similar_initiatives
```

Comment:
Also I know this is 100% cosmetic and should be caught by rubocop, but can we add an empty line here? 😇 
```suggestion
      helper SignatureTypeOptionsHelper

      helper_method :similar_initiatives
```